mesh/meshdb.go

```diff
@@ -185,7 +185,7 @@ func (m *meshDB) handleLayerBlocks(ll *layerHandler) {
 
 			if err := m.blocks.Put(bl.ID().ToBytes(), bytes); err != nil {
 				log.Error("could not add bl to ", bl.ID(), " database ", err)
-				continue
+				return
```

Comment:
panic here, this should not happen, if we cant insert into DB we cant rely on our network image in meshdb anymore